test/Microsoft.AspNet.Mvc.FunctionalTests/WebApiCompatShimBasicTest.cs

```diff
@@ -19,21 +19,25 @@
 
 namespace Microsoft.AspNet.Mvc.FunctionalTests
 {
-    public class WebApiCompatShimBasicTest
+    public class WebApiCompatShimBasicTest : IClassFixture<MvcTestFixture<WebApiCompatShimWebSite.Startup>>
     {
         private const string SiteName = nameof(WebApiCompatShimWebSite);
         private readonly Action<IApplicationBuilder> _app = new WebApiCompatShimWebSite.Startup().Configure;
```

Comment:
A bug I haven't got to the bottom of (specific to Core CLR test runs) prevents changing 3 test methods in this class.